examples/bri-3/src/bri/identity/bpiAccounts/agents/bpiAccountsStorage.agent.ts

```diff
@@ -26,6 +26,8 @@ export class BpiAccountStorageAgent extends PrismaService {
             ownerBpiSubject: true,
           },
         },
+        // look into merkle tree profile to resolve unmarhsling of the state tree here
```

Comment:
@ognjenkurtic do you want to list that as an open issue or close out before the PR is merged? At least, list as a TODO comment, please.